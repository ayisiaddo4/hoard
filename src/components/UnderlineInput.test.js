/* eslint-env jest */

import React from 'react';
import UnderlineInput from './UnderlineInput';

import renderer from 'react-test-renderer';

const onChangeText = jest.fn();

const MockedComponent = () => (
  <UnderlineInput label="Label" onChangeText={onChangeText} />
);

test('renders correctly', () => {
  const tree = renderer.create(<MockedComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
