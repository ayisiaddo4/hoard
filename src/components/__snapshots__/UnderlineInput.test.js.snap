// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityLabel="Label"
  accessible={true}
  style={
    Array [
      Object {
        "marginBottom": 10,
        "marginTop": 10,
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    collapsable={undefined}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "rgba(255, 255, 255, 0.502)",
        "fontSize": 14,
        "left": 0,
        "position": "absolute",
        "top": -20,
      }
    }
  >
    Label
  </Text>
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    editable={true}
    keyboardType="default"
    label="Label"
    onBlur={[Function]}
    onChangeText={[MockFunction]}
    onEndEditing={undefined}
    onFocus={[Function]}
    onSubmitEditing={[Function]}
    returnKeyType="done"
    secureTextEntry={false}
    style={
      Array [
        Object {
          "borderBottomWidth": 1,
          "height": 40,
        },
        Object {
          "color": "#fff",
        },
        Object {
          "borderBottomColor": "rgba(255,255,255, 0.25)",
        },
        Object {},
      ]
    }
    underlineColorAndroid="transparent"
    value={undefined}
  />
</View>
`;
